### ISSUE TITLE
Update svelte-jsoneditor to Latest

### ISSUE THREAD

**Issue by ptavarez** [ballercat/jambox#61]
### Description
Hello again!

Thank you for your quick response to the previous issue regarding the Svelte version update. We appreciate your efforts in resolving that dependency.

Unfortunately, we're still encountering compatibility issues with Storybook 8 because the current version of `svelte-jsoneditor` used by JamBox relies on Svelte 3. To fully resolve the issue, we need `svelte-jsoneditor` to be updated to a version that uses Svelte 4 (0.23.8 should do).

### Error
![Screenshot 2024-08-01 at 4 59 34 PM](https://github.com/user-attachments/assets/f64c0c14-a21a-45a6-8a03-2633b3209422)

### Request:

Please update the `svelte-jsoneditor` dependency to a version that uses Svelte 4 in JamBox. This will ensure compatibility with Storybook 8 and align all dependencies with Svelte 4.

Apologies for not being clearer about the specific requirements initially. Thank you for your continued support and assistance in this matter.

**Comment by ballercat** [comment]
Yeah so this is a much bigger effort because upgrading that dep no longer builds the dist due to webpack errors. I'd have to sped quite a bit of time chasing those down. Maybe even change build tooling...

To be honest I'm not sure how exactly this is blocking you from upgrading storybook. Do you actually use svelte in storybook yourself? Why can't you pin your own svelte dependency? 

Best I can do for now is move the svelte related dependencies to `devDependencies` since they are not used at runtime.

**Comment by ballercat** [comment]
@ptavarez you can try giving version `0.4.3` a shot which has svelte dependencies moved to `devDependencies`

Going to leave this open as I'd like to actually do this upgrade at some point...
